mentat/conversation.py

```diff
@@ -18,13 +23,17 @@
     is_model_available,
     model_context_size,
 )
-from .parsers.file_edit import FileEdit
-from .parsers.parser import PARSER, Parser
-from .session_stream import SESSION_STREAM, SessionStream
+from .session_stream import SESSION_STREAM
 
 CONVERSATION: ContextVar[Conversation] = ContextVar("mentat:conversation")
 
 
+class MessageRole(Enum):
```

Comment:
I think it's better to call this `Role`. Thanks for refactoring it in this way. I think it's a bit cleaner.